src/include/network/network_defs.h

```diff
@@ -115,6 +115,7 @@ enum class QueryType : uint8_t {
   QUERY_DROP_TRIGGER,
   QUERY_DROP_SCHEMA,
   QUERY_DROP_VIEW,
+  QUERY_EXPLAIN,
```

Comment:
Do we believe `EXPLAIN` should fall into non-transactional instead? I guess I can test this.

Reply:
Yes, it should be non-transactional because since SQL EXPLAIN just generates the query plan, it does not actually modify or access the database and thus I think it being non-transactional is fine.